pages/models.py

```diff
@@ -77,24 +77,41 @@ class RootPage(Page):
         ('hero', blocks.StructBlock([
             ('text', APIRichTextBlock()),
             ('image', APIImageChooserBlock(required=False)),
-            ('cta', blocks.ListBlock(CTAButtonBlock(required=False), max_num=2, label='Calls to Action')),
+            ('cta', blocks.ListBlock(CTALinkBlock(required=False, label="Button"),
```

Comment:
I think you are changing the label here back to "Button", do you want this to be "Calls to Action"?
https://github.com/openstax/openstax-cms/pull/1567/commits/6d81c65866b19e343dd208ee7f92a113e75faac8#diff-0632a78c257031516b92ee8143b274cd6af0c247ebdc1c2f5eadefa2036179baR58